src/Config/HotReloadEventHandler.cs

```diff
@@ -22,14 +22,13 @@ public HotReloadEventHandler()
     {
         _eventHandlers = new Dictionary<string, EventHandler<TEventArgs>?>
         {
+            // QueryManagerFactory will create a new set of QueryExecutors
+            // during Hot-Reload so it is not necessary to specifically reconfigure
+            // QueryExecutors as part of the Hot-Reload process.
```

Comment:
below this comment, ensure that there is consistent spacing between string constants and null. e.g. the following are inconsistent with the rest:
```csharp
{ QUERY_ENGINE_FACTORY_ON_CONFIG_CHANGED,null },
{ MUTATION_ENGINE_FACTORY_ON_CONFIG_CHANGED,null },
```

Reply:
i swear sometimes VS just randomly changes my formatting when I copy paste things. Fixed.